cmd/stacker/internal_go.go

```diff
@@ -61,6 +65,18 @@ var internalGoCmd = cli.Command{
 				},
 			},
 		},
+		&cli.Command{
+			Name:   "bom-discover",
+			Action: doBomDiscover,
+		},
+		&cli.Command{
+			Name:   "bom-build",
+			Action: doBomBuild,
+		},
+		&cli.Command{
+			Name:   "bom-verify",
+			Action: doBomVerify,
```

Comment:
We don't have any convention in stacker here yet as to whether we want:

sub-subcommands (stacker bom build) or hyphenated commands (stacker bom-build).

because we're going to expose that and even document it to the user as the way to generate bom information, we should at least think on it.